mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -521,7 +545,7 @@ properties:
           - name: 'customRepository'
             type: NestedObject
             description: |-
-              Settings for a remote repository with a custom uri.
+              [Deprecated, please use commonRepository instead] Settings for a remote repository with a custom uri.
```

Comment:
Just want to make sure, the common_repository are well-tested and stable enough in the probe results that we can safely ask users to use it ove the old configs?

Reply:
They are already used everywhere by UI and gcloud instead of Custom repositories.